dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/interceptor/JAXRSOutInterceptor.java

```diff
@@ -232,7 +232,10 @@ private void serializeMessage(ServerProviderFactory providerFactory,
             }
             responseMediaType = checkFinalContentType(responseMediaType, writers, checkWriters);
         } catch (Throwable ex) {
-            handleWriteException(providerFactory, message, ex, firstTry);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, ex.getMessage() + ", " + ex);
+            }
+			handleWriteException(providerFactory, message, ex, firstTry);
```

Comment:
Nit, but there are extra tabs/white space indenting the `handleWriteException(...)` call.

Reply:
Done